zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -1012,7 +1012,28 @@ public void updatePersonalizedMode(String noteId,
   public void moveNoteToTrash(String noteId,
```

Comment:
Besides moving to trash, can corrupted note be deleted from trash ?

Reply:
No, but have already fixed in the last commit.